folsom/src/main/java/com/spotify/folsom/client/DefaultRawMemcacheClient.java

```diff
@@ -300,12 +301,13 @@ public int numActiveConnections() {
                       return;
                     }
                     if (timeoutChecker.check(head)) {
-                      log.error("Request timeout: {} {}", channel, head);
+                      double elapsedMs = timeoutChecker.elapsedNanos(head) / NANOS_IN_MILLIS;
```

Comment:
Could the `check()` method return the elapsed nanos directly instead of calculating elapses nanos twice?

Do we need this double division? Can we just log the nanos instead or go with a cheaper integer `TimeUnit.NANOSECONDS.toMillis()`?